test/shm_size.bats

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bats
+
+load helpers
+
+function setup() {
+	setup_test
+}
+
+function teardown() {
+	cleanup_test
+}
+
+@test "check /dev/shm is changed" {
+	start_crio
+	replace_config "enable_custom_shm_size" "true"
```

Comment:
ah, replace_config is not the right approach here. Instead, you should use the environment variable `CONTAINER_ENABLE_CUSTOM_SHM_SIZE` and set it as such before the `start_crio` call as in
`CONTAINER_ENABLE_CUSTOM_SHM_SIZE=true start_crio`